client/html/templates/catalog/detail/service-body-standard.php

```diff
@@ -11,9 +11,9 @@
 ?>
 <?php $this->block()->start( 'catalog/detail/service' ); ?>
 <div class="catalog-detail-service">
-	<span class="service-intro"><?= $enc->html( $this->translate( 'client', '+ shipping costs' ) ); ?></span>
 
 	<?php if( !empty( $items ) ) : ?>
+	<span class="service-intro"><?= $enc->html( $this->translate( 'client', '+ shipping costs' ) ); ?></span>
```

Comment:
Can you please correct the indention (2 tabs)?

Reply:
done